src/Explorer/Explorer.tsx

```diff
@@ -1178,7 +1179,11 @@ export default class Explorer {
   }
 
   public async configureCopilot(): Promise<void> {
-    if (userContext.apiType !== "SQL" || !userContext.subscriptionId) {
+    if (
+      userContext.apiType !== "SQL" ||
+      !userContext.subscriptionId ||
+      [Environment.Development, Environment.Mpac, Environment.Prod].indexOf(getEnvironment()) === -1
```

Comment:
I think
![values].includes(getEnvironment())
is easier to understand at a glance